### ISSUE TITLE
BUG website seems down

### ISSUE THREAD

**Issue by mathurinm** [scikit-learn-contrib/scikit-learn-contrib#70]
sklearn-contrib hosted documentaitons such as skglm  or metric-learn (https://contrib.scikit-learn.org/skglm/, https://contrib.scikit-learn.org/metric-learn/) now return a 404 error

**Comment by rth** [comment]
I saw a notification this weekend,

> ogrisel has transferred the scikit-learn-contrib/scikit-learn-contrib.github.io repository to scikit-learn/scikit-learn-contrib.github.io.

Can you contact @ogrisel about it ?

**Comment by rth** [comment]
And also there was,

![Image](https://github.com/user-attachments/assets/f4632857-4f9c-4f81-82f0-4dea1e3f3643)

not exactly sure how org wise github pages works anymore..

**Comment by rth** [comment]
So I re-aded the contrib.scikit-learn.org domain to  in https://github.com/organizations/scikit-learn-contrib/settings/pages

But someone who has access to the DNS configuration of scikit-learn.org would need to re-validate it
```
1. Create a TXT record in your DNS configuration for the following hostname: _github-pages-challenge-scikit-learn-contrib.contrib.scikit-learn.org
2. Use this code for the value of the TXT record: 4b01ab915a1f3012cd0d01a9fe700c
3. Wait until your DNS configuration changes. This could take up to 24 hours to propagate.
```
cc @glemaitre

**Comment by ogrisel** [comment]
@lesteve activated github domain verification to protect the main scikit-learn.org website but we did not anticipate the impact on subdomains. The https://contrib.scikir-learn.org readme redirection was fixed by moving its repo to the scikit-learn organization.

I thought that individual projects used their own domain names and that should be fine but i did not anticipate that some relied on the contrib.scikit-learn.org cname...

@lesteve @betatim shall we disable the verified domain protection for the main website?

**Comment by rth** [comment]
Makes sense. Are you sure if you just re-add the subdomain (what I did above) and re-validate it, while keeping the main domain protected it won't work?

**Comment by ogrisel** [comment]
No: we tried to verify `contrib.scikit-learn.org` explicitly with its own `TXT` DNS record from the github.com/scikit-learn-contrib organization settings, but it does not work. I think it's not possible, to verify the `scikit-learn.org` on the `scikit-learn` org and still continue using the `contrib.scikit-learn.org` CNAME for repos hosted in other orgs as explained in:

https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages#about-domain-verification-for-github-pages

**Comment by ogrisel** [comment]
I wanted to temporarily undo the verified domain config in the `scikit-learn` org settings but I don't understand how to do it.

So instead, as a temporary fix, I changed the CNAME config for `contrib.scikit-learn.org` at our DNS provider to instead configure https://contrib.scikit-learn.org/anything to be temporarily (web) forwarded to https://scikit-learn-contrib.github.io/anything.

That it not as cool as using the scikit-learn domain, but that should hopefully temporarily fix broken links while we devise a better long-term solution.

**Comment by ogrisel** [comment]
The DNS config propagation might take a while (typically minutes to a few hours) though. So please be patient.

**Comment by ogrisel** [comment]
The redirection seems to work when I use a browser I had not used in a while.

**Comment by rth** [comment]
Thanks for looking into it @ogrisel !

**Comment by ogrisel** [comment]
> No: we tried to verify contrib.scikit-learn.org explicitly with its own TXT DNS record from the github.com/scikit-learn-contrib organization settings, but it does not work. I think it's not possible, to verify the scikit-learn.org on the scikit-learn org and still continue using the contrib.scikit-learn.org CNAME for repos hosted in other orgs as explained in [...]

Actually, my prior attempt might not have worked because I mistakenly confused 

https://github.com/organizations/scikit-learn-contrib/settings/domains

for 

https://github.com/organizations/scikit-learn-contrib/settings/pages

But they serve different purposes.

Let me try to actually verify contrib.scikit-learn.org from the `scikit-learn-contrib` org and see if I can reconfigure the CNAMEs as originally (by ending the web forwarding from contrib.scikit-learn.org to scikit-learn-contrib.github.io).

If that does not work I will put back the webforwarding in place.

**Comment by ogrisel** [comment]
After anxiously waiting a while of DNS / TLS config/ cache propagation throughout the Github servers, I think all is working again as it used to work now and we have enabled verified domains both for scikit-learn.org and contrib.scikit-learn.org.

Please feel free to reopen if stuff are still broken for you.

**Comment by ogrisel** [comment]
Sorry for the disruption and thank for the report/ping @mathurinm @rth.

**Comment by rth** [comment]
Thanks a lot @ogrisel
